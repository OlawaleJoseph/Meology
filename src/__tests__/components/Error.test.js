import React from 'react';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom';
import { createMemoryHistory } from 'history';
import ErrorPage from '../../pages/Error';

describe('Home Page Snapshot', () => {
  it('builds home page correctly with search and meallist', () => {
    const tree = renderer.create(
      <Router history={createMemoryHistory()}>
        <ErrorPage />
      </Router>,
    );

    expect(tree).toMatchSnapshot();
  });
});
