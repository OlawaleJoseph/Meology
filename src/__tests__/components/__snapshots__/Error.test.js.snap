// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Snapshot builds home page correctly with search and meallist 1`] = `
<section
  className="error-page section"
>
  <div
    className="error-container"
  >
    <h1>
      oops! it's a dead end
    </h1>
    <a
      className="btn btn-primary"
      href="/"
      onClick={[Function]}
    >
      back home
    </a>
  </div>
</section>
`;
